docs/DeveloperGuide.adoc

```diff
@@ -439,7 +439,7 @@ The sort enhancement utilises the Java Collections Sort API by passing it a cust
 The sort command is parsed through `SortCommandParser`, which passes control over to the `SortCommand`
 class. The actual sorting happens via the UniquePersonList class.
 
-The Logic Diagram for this sort command implementation is shown below:
+The UML outline diagram for this sort command implementation is shown below:
```

Comment:
Never heard of `UML outline diagrams` before. :-p

Reply:
XD I'll change it to UML diagram. "Outline" matches the objective of the diagram though.

Reply:
`UML diagram` is too general. Identify what kind of a diagram it is. e.g. `UML ABC diagram showing classes relevant to the sort command`

Reply:
Thanks for the advice! I will look into it!